### DISCUSSION TITLE
Person Entity to Map View?

### DISCUSSION THREAD

**Discussion by TheOpsMgr** [matt8707/ha-fusion#415] [Q&A]
In the YouTube overview it shows clicking a Person entity and that opens up a map view.

I cant get that to work that way - If I click it just get a view of the person entity state. 

![image](https://github.com/matt8707/ha-fusion/assets/3353242/7498af0b-5240-4373-bd12-c42155d40bb7)

If I change the Entity to the related person tracker, which returns the gps location, THEN I get a map view as per the demo.

I can't see anything in the demo yaml that is using templating to extract the device_tracker and location from the person entity, so not sure how this is supposed to work as demo'd.

Note I am still pretty new to HA and have very little templating experience in HA, I'm mostly just using GUI stuff at the moment, so it's quite possible I missed something obvious.

**Comment by cru5h3r** [comment]
Hi Steve,
How about using just the "person tracker entity" instead of the "person entity"?
Just change the icon of the "person tracker entity" using this code:

```
{% if is_state(entity_id, "on") %}
  /local/your_picture_inside_folder_www.jpg
{% else %}
  /local/your_picture_inside_folder_www.jpg
{% endif %}
```

That's what I did here
